src/ghdcbot/core/social_models.py

```
@@ -0,0 +1,105 @@
+
+
+class LinkedInProfile(BaseModel):
+    """Normalized LinkedIn profile"""
+    profile_url: str  # Full normalized URL
+    profile_id: str   # Extracted profile ID
+    
+    @field_validator("profile_url")
+    @classmethod
+    def validate_url(cls, value: str) -> str:
+        if not value.startswith("https://"):
+            raise ValueError("LinkedIn URL must start with https://")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)